src/lib/crypto/kimchi_backend/common/bigint.ml

```diff
@@ -66,12 +70,10 @@ module Make
 
   let sexp_of_t t = to_hex_string t |> Sexp.of_string
 
-  let of_hex_string ?(reverse = true) s =
+  let of_hex_string s =
     assert (Char.equal s.[0] '0' && Char.equal s.[1] 'x') ;
     let s = String.drop_prefix s 2 in
-    Option.try_with (fun () -> Hex.decode ~init:Bytes.init ~reverse s)
-    |> Option.value_exn ~here:[%here]
-    |> of_bytes
+    of_hex s
```

Comment:
* `reverse` was never used in practice
* it's crazy anyway to support two byte ordering for serialization, let's choose one canonical one (arkworks settled on LE) and use it everywhere

Reply:
This changes the GraphQL user interface. Revert

Reply:
OK I'm thinking of just changing our `to_hex` functions within proof systems to emit BE encoding, and use that everywhere, wdyt?